packages/cozy-harvest-lib/src/components/AccountForm/index.jsx

```diff
@@ -193,13 +207,16 @@ export class AccountForm extends PureComponent {
       onSubmit,
       showError,
       t,
+      fieldsOptions,
       flowState
     } = this.props
     const submitting = flowState.running
     const error = flowState.error
 
     const { fields } = konnector
     const sanitizedFields = manifest.sanitizeFields(fields)
+    manifest.addForceEncryptedPlaceholder(sanitizedFields, fieldsOptions)
```

Comment:
The forceEncryptedPlaceholder option is not supported by the manifest, only by the UI fields, so I would not put the helper function in the `manifest` file.

Reply:
Move to fields helper